src/docs/index.jsx

```diff
@@ -561,51 +550,57 @@ const App = createClass({
 
 	searchResults() {
 		const { search } = this.state;
-
-		return _.flatMap(docgenMap, (value, componentName) => {
-			if (!this.showPrivateComponents() && value.isPrivateComponent) {
-				return [];
-			}
-
-			// exclude child components from search
-			if (_.includes(componentName, '.')) {
-				return [];
-			}
-
-			if (componentName.toLowerCase().indexOf(search.toLowerCase()) !== -1) {
-				return componentName;
+		const searchLower = _.toLower(search);
+		const showPrivateComponents = this.showPrivateComponents();
+
+		return _.filter(_.keys(lucidComponents), key => {
+			const componentRef = lucidComponents[key];
+			const isPrivate =
+				_.get(componentRef, 'peek.isPrivate') ||
```

Comment:
How does peek map to private? would private be set on a prop?

Reply:
the `isPrivate` flag is set on proptypes resolvers which are accessed from the `.isRequired` reference of the normal resolver.